get_started.sh

```diff
@@ -26,6 +26,8 @@ SHELL_CONFIG_FILES=(
 declare -a new_shell_config_lines=(
     # Source the ROS Environment Variables Automatically
     "source /opt/ros/kinetic/setup.sh"\
+    # Setup for rosinstall, sources env setup of packages installed through .rosinstall
+    "source $DIR/extended_pkg/setup.sh"
```

Comment:
this should have a `\` character at the end I think? If we don't need it for this line (ie. it works without) we should remove it from the other lines in `new_shell_config_files` as well.

Reply:
`get_started.sh` no longer exists